package.json

```diff
@@ -229,6 +229,7 @@
     "terser": "5.25.0",
     "tslib": "2.6.2",
     "typescript": "5.3.2",
+    "webdriverio": "^8.26.1",
```

Comment:
```suggestion
    "webdriverio": "8.26.1",
```

Reply:
We can't use suggestion to update this as it does not update the `package-lock.json`.